PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -206,7 +221,7 @@
         LepGood_sip3d = cms.string("abs(dB('PV3D')/edB('PV3D'))"),
         LepGood_dxy = cms.string("log(abs(dB('PV2D')))"),
         LepGood_dz = cms.string("log(abs(dB('PVDZ')))"),
-        LepGood_mvaIdFall17noIso = cms.string("userFloat('mvaFall17noIso')"),
+        LepGood_mvaIdFall17noIso = cms.string("userFloat('mvaFall17V1noIso')"),
```

Comment:
Yes, because here our Egamma MVA is used as an input variable for the ttH MVA, which I'm not responsible for. Someone else should eventually decide if the version there should be bumped.